### PR TITLE
[Merged by Bors] - Add pixelated Bevy to assets and an example

### LINKED ISSUE #2279: Add pixelated Bevy to assets
@dtcristo, the author of [bevy_pixels](https://github.com/dtcristo/bevy_pixels) created the awesome little pixeled Bevy shown below.

To avoid confusion with the official project, they've offered to it to Bevy to use:
> Feel free to use it for whatever official purposes you want. I created it using https://www.pixilart.com/

- [@dtcristo on Discord](https://discord.com/channels/691052431525675048/743664267856838736/848379623943045173)

As we progress on #1856, it would be valuable to have test assets of this sort, and this is both small *and* cute.
I've attached the image and .pixil file below.

[bevy_pixil.zip](https://github.com/bevyengine/bevy/files/6565567/bevy_pixil.zip)
bevy_pixels

### PR DESCRIPTION
**Pull request by Ian-Yy**
# Objective
Fixes #2279 

## Solution
Added pixelated Bevy to assets folder and used in a `pixel_perfect` example.

### CODE REVIEW

File: examples/2d/pixel_perfect.rs [hunk 1]
```diff
@@ -0,0 +1,45 @@
+//! Renders a 2D scene containing pixelated bevy logo in pixel perfect
+
+use bevy::prelude::*;
+
+fn main(){
+    App::new()
+        .add_plugins(DefaultPlugins)
+        .add_plugin(ImagePlugin::default_nearest())
```
Comment (mockersf) [inline examples/2d/pixel_perfect.rs:8]:
```suggestion
        .add_plugins(DefaultPlugins.set(ImagePlugin::default_nearest()))
```

**Comment by alice-i-cecile** [comment]
@JonahPlusPlus, what do you want to do for the license here? My instincts say that the pixelated logo should be available under the same terms as the main logo, which IIRC now means "copyright assignment to @cart, who then gives permission to people to use it in splash screens but not to misrepresent their relationship to Bevy".

File: examples/2d/pixel_perfect.rs [hunk 2]
```diff
@@ -0,0 +1,45 @@
+//! Renders a 2D scene containing pixelated bevy logo in pixel perfect
```
Comment (alice-i-cecile) [inline examples/2d/pixel_perfect.rs:1]:
```suggestion
//! Renders a 2D scene containing pixelated bevy logo in a pixel perfect style
```

**Comment by alice-i-cecile** [comment]
(Controversial label added just due to licensing).

**Comment by JonahPlusPlus** [comment]
@alice-i-cecile Sure, that works. I was fine with it being CC0, but I totally get why it's better for it owned by cart.

**Review by alice-i-cecile — APPROVED** [review]

**Comment by james7132** [comment]
bors try

**Review by james7132 — APPROVED** [review]

**Pull request closed without merging**